### PR TITLE
Use Pandora logo if the album art url is no good.

### PR DESCRIPTION
**Pull request by JasonLG1979**
1st choice is of course the actual album cover. 2nd choice is a generic Pandora logo and 3rd choice is the default blue gradient.

### CODE REVIEW

File: pithos/pithos.py [hunk 1]
```diff
@@ -56,6 +56,7 @@
 
 ALBUM_ART_SIZE = 96
 ALBUM_ART_X_PAD = 6
+GENERIC_LOGO = 'http://cont-sv5-1.pandora.com/images/public/devicead/1/n/r/a/daarcmdywarn1_500W_500H.jpg'
```
Comment (TingPing) [inline pithos/pithos.py]: This looks like an auto generated URL I would not trust existing for long.

**Comment by JasonLG1979** [comment]
http://cont-sv5-1.pandora.com/images/public/devicead/1/n/r/a/daarcmdywarn1_500W_500H.jpg
Still good.

It would have to be updated if it were ever to become no good.

**Comment by JasonLG1979** [comment]
It's the url that's served up with the multiple stream warning.

**Comment by TingPing** [comment]
That url doesn't even work for me...

**Comment by JasonLG1979** [comment]
Funny works fine for me?

**Comment by JasonLG1979** [comment]
screenshot from 2015-05-22 14 46 19

**Comment by JasonLG1979** [comment]
screenshot from 2015-05-22 14 49 06

**Comment by TingPing** [comment]
Hmm, it works in another browser but not chrome. Anyway I still feel funny about the URL and...

> It would have to be updated if it were ever to become no good.

Is not valid for any project with a release cycle. There is no point in adding it if it may not work for 80% of our users in 6mo.

**Comment by JasonLG1979** [comment]
> Is not valid for any project with a release cycle. There is no point in adding it if it may not work for 80% of our users in 6mo.

Fair point. Just trying to find something better than the gradient. I'll see if I can't find something from Pandora that is legally OK to use.

**Comment by JasonLG1979** [comment]
Never mind can't ship there logo in the app according to their brand guide without written consent. I was trying to side step that by just basically linking to a publicly accessible url. But never mind I guess. Would you receptive to either, 1. Using a default media icon. 2. A redesign of the "default_icon" image or 3. Using the Pithos logo maybe? If none of those thing sound like they're worth it, no big deal.

**Comment by TingPing** [comment]
I am not in love with the current fallback icon so any changes you have in mind might be interesting. I don't think the pithos logo would fit in though.

File: pithos/pithos.py (hunk 1 above)
Comment (JasonLG1979) [inline pithos/pithos.py]: It still exists,lol!!!

**Pull request closed without merging**